src/libraries/System.Net.Http/src/System.Net.Http.csproj

```diff
@@ -18,6 +18,8 @@
   </PropertyGroup>
   <ItemGroup>
     <ILLinkSubstitutionsXmls Include="$(ILLinkDirectory)ILLink.Substitutions.xml" />
+    <ILLinkSubstitutionsXmls Condition="'$(TargetsAndroid)' == 'true' or '$(TargetsiOS)' == 'true' or '$(TargetsMacCatalyst)' == 'true' or '$(TargetsTVOS)' == 'true'"
```

Comment:
nit: we can simplify it to just use `'$(TargetsMobile)' == 'true'` here which is defined at repo's [root](https://github.com/dotnet/runtime/blob/51886d1b56eed00dcde92a2efc448071c1cf49b6/Directory.Build.props#L22).

Reply:
The issue with `TargetsMobile` is this clause:

```
or '$(TargetOS)' == 'Browser'
```

https://github.com/dotnet/runtime/blob/51886d1b56eed00dcde92a2efc448071c1cf49b6/Directory.Build.props#L22

Reply:
Ah, so we have more than one definitions of isMobile 😁 I think we can still can use shorter syntax and exclude the browser.